### ISSUE TITLE
ShiftToStartIMU(float pointTime) is not used

### ISSUE THREAD

**Issue by deliangye** [RobustFieldAutonomyLab/LeGO-LOAM#7]
imuShiftFromStartXCur, imuShiftFromStartYCur, imuShiftFromStartZCur always set to 0, since the function ShiftToStartIMU is not called. right?

**Comment by TixiaoShan** [comment]
@deliangye 
Yes, it is not used. I encountered very bad drift after enabling it.
